frontend/src/scenes/billing/Billing.tsx

```diff
@@ -63,29 +65,24 @@ export function Billing(): JSX.Element {
     }
 
     if (!billing && !billingLoading) {
-        const supportLink = (
-            <Link
-                target="blank"
-                to="https://posthog.com/support?utm_medium=in-product&utm_campaign=billing-service-unreachable"
-            >
-                {' '}
-                contact support{' '}
-            </Link>
-        )
         return (
             <div className="space-y-4">
                 <BillingPageHeader />
                 <LemonBanner type="error">
                     There was an issue retrieving your current billing information. If this message persists please
-                    {supportLink}.
+                    <LemonButton
```

Comment:
Definitely want to use a link here and not a button so that it's inline. Links don't have to link out, they can take `onclick` props.